pkg/rancher-desktop/assets/scripts/service-k3s.initd

```diff
@@ -39,10 +39,19 @@ start_pre() {
   if [ "$(uname -m)" = "aarch64" ]; then
     ARCH=arm64
   fi
+  IMAGEPATH="/var/lib/rancher/k3s/agent/images/k3s-airgap-images-${ARCH}"
   if [ "${ENGINE}" == "moby" ]; then
-    docker load --input /var/lib/rancher/k3s/agent/images/k3s-airgap-images-${ARCH}.tar
+    if [ -f ${IMAGEPATH}.tar.zst ]; then
+      zstd -f -c -d ${IMAGEPATH}.tar.zst | docker load
+    else
```

Comment:
I know we should not start the `k3s` service unless on of the airgap tarballs exist, but it feels more robust to check if the `*.tar` file exists before trying to load it:

```suggestion
    elif  [ -f ${IMAGEPATH}.tar ]; then
```

And same thing in the `else` branch below.

Reply:
I made the suggested change.  Initially, I did not add the 'if' block around the second files to avoid changing the behavior of the script.  But my testing shows that with the new change the script fails silently if the files don't exist, rather than crashing which allows a later process to attempt to download the image.  This is a change in behavior, but it seems to be an improvement.